app/src/pages/Lobby/Inputs.tsx

```diff
@@ -179,7 +166,6 @@ export function SubmissionsPerPlayerInput(props: { value: string }) {
       helperText={
         <HelperText>Depends on group size, but usually 4-6</HelperText>
       }
-      defaultValue={props.value}
```

Comment:
+1 agree we don't need this given that we set textFieldValue based on props anyways, as a side effect. related to https://reactjs.org/docs/uncontrolled-components.html